scripts/queuestat

```diff
@@ -346,8 +335,10 @@ class Queuestat(object):
         for port in natsorted(self.counter_port_name_map):
             json_output[port] = {}
             cnstat_dict = self.get_cnstat(self.port_queues_map[port])
-
-            cnstat_fqn_file_name = cnstat_fqn_file + port
+            cache_ns = ''
+            if self.voq and self.namespace is not None:
```

Comment:
We should also handle cases when namespace arg is provided but the voq flag is absent